### ISSUE TITLE
multiple signatures and countersigning

### ISSUE THREAD

**Issue by nordoff** [rauc/rauc#705]
Hi,
How could I accomplish multiple signatures and countersigning of bundles?
I have a use case where I want to use countersignatures so users can control which updates may be installed. Also, I want to support "m of n" signatures to mitigate key compromise on my side. For example, I could have a bundle signed with 3 signatures, and the customer could then countersign those signatures (1 or more times).

I see that CMS is used for handling bundle signing, and I believe the format supports both of these cases so it seems this should be possible.

**Comment by jluebbe** [comment]
Yes, this feature was one of the reasons why we choose CMS. So far, adding signatures or n-of-m checking during verification is not implemented. We'd be happy to review and accept contributions, though.
